app/views/homepage/index.html.erb

```diff
@@ -63,7 +63,7 @@
             </li>
             <li>
               <a href="/government/organisations#agencies_and_other_public_bodies" class="home-numbers__link govuk-link">
-                <strong class="home-numbers__large">412</strong>
+                <strong class="home-numbers__large">400+</strong>
```

Comment:
Do you reckon there's any potential benefit to putting this in a translation file?